index.html

```diff
@@ -185,17 +185,11 @@ <h1 id="images-and-videos">Images and videos</h1>
 &lt;/section&gt;</code></pre>
             <figcaption>
               <h1>Code snippet of an image, with a descriptive caption.</h1>
-              <p>The <code>&lt;figure&gt;</code> element may be avoided if there is no need for a caption, in such a case the <code>&lt;img&gt;</code> element is a child of the <code>&lt;section&gt;</code> element. The article header may contain a picture as well, which will be full width.</p>
+              <p>The <code>&lt;figure&gt;</code> element may be avoided if there is no need for a caption, in such a case the <code>&lt;img&gt;</code> element is a child of the <code>&lt;section&gt;</code> element. The default width is 600px, authors can use inline style declarations to customise the width, as described in section <strong><a href="#customisations">Customisations</a></strong>.</p>
```

Comment:
Updated, let me knows if it is clearer:

> The default width of images is 600px, equals to the default width of text paragraph and code snippets.

I did not use the word "body" because it is an HTML tag, so it may be misleading.